docs/Versioning.md

```diff
@@ -7,18 +7,16 @@ specification, MUST, SHOULD et al are used consistent with [RFC2119](https://too
 
 ONNX defines versioning policy and mechanism for three classes of entities:
 
-* The abstract graph model and the concrete format that serializes it. The graph model and format are versioned atomically and are referred to as the *IR version.* The IR version is  represented by the `ModelProto.ir_version` field.
-* Operators that are referenced by a given ONNX graph. The version of a given operator  is referred to as the *operator version*. The operator version is  represented by the `TBD` field.
-* An ONNX ModelProto that represents a given graph - that is, the contents of a model. We refer to this version as the *model version* and it is represented by the `ModelProto.model_version` field.
+* The abstract model for graphs and operators and the concrete format that represents them. These are always versioned atomically and are referred to as the *IR version.* 
```

Comment:
"The abstract model for graphs and operators ...",  operators are not covered in "IR version" now, I think. Am I right?